xmtp/src/builder.rs

```diff
@@ -33,14 +36,47 @@ where
         self
     }
 
+    pub fn account(mut self, wallet_address: String) -> Result<Self, String> {
```

Comment:
Maybe should rename this to `find_or_create_account`?

Reply:
down for that name!